dependency-check-suppression.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
+  <suppress>
+    <notes><![CDATA[This is a JS dependency.]]></notes>
+    <cve>CVE-2021-41251</cve>
```

Comment:
(Major/Question)

How exactly is the JS dependency recognized?
If we are affected by this false-positive(?) then all other users of "owasp" will have a similar experience, right?

Reply:
It linked me to a dependency called `cloud-sdk-js@core 1.0`. I guessed it saw our openAI module use `com.sap.ai.sdk:core:0.1.0-SNAPSHOT` and thought it was good enough of a match...
I don't think this plugin is great, just better than blackduck hopefully.